addons/pos_restaurant/static/tests/tours/helpers/BillScreenTourMethods.js

```diff
@@ -18,3 +18,42 @@ export function isShown() {
         },
     ];
 }
+
+export function clickBillButton() {
+    return [
+        {
+            content: "click review button",
+            trigger: ".btn-switchpane.review-button",
+            mobile: true,
+        },
+        {
+            content: "click more button",
+            trigger: ".mobile-more-button",
+            mobile: true,
+        },
+        {
+            content: "click bill button",
+            trigger: '.control-button:contains("Bill")',
+        },
+    ];
+}
+
+export function isQRCodeShown() {
+    return [
+        {
+            content: "QR codes are shown",
+            trigger: '#posqrcode',
+            run: () => {},
+        },
+    ];
+}
+
+export function isQRCodeNotShown() {
+    return [
+        {
+            content: "QR codes are not shown",
+            trigger: 'body:not(:has(#posqrcode))',
+            run: () => {},
+        },
+    ];
+}
```

Comment:
can you use `negateStep()` from `/point_of_sale/static/tests/tours/helpers/utils.js` instead of creating a new method?